### PR TITLE
Pin the Saturation and Value parameters in ShadowColor#HSVToColor

### PR DESCRIPTION
**Pull request by mlebedy**
### Overview

Fix for "Color.HSVToColor method parameters should be pinned" #8275

### CODE REVIEW

**Comment by utzcoz** [comment]
@mlebedy please squash commits to one.

**Comment by utzcoz** [comment]
@mlebedy You also need to follow https://github.com/robolectric/robolectric/wiki/Robolectric's-code-style to format your code style.

File: robolectric/src/test/java/org/robolectric/shadows/ShadowColorTest.java [hunk 1]
```diff
@@ -74,4 +74,16 @@ public void HSVToColorShouldReverseColorToHSV() {
...
+
+  @Test
+  public void HSVToColorValueShouldBePinned() {
+    assertThat(Color.HSVToColor(new float[]{0f, 0f, -1.0f})).isEqualTo(Color.BLACK);
+    assertThat(Color.HSVToColor(new float[]{0f, 0f, 2.0f})).isEqualTo(Color.WHITE);
+  }
+
+  @Test
+  public void HSVToColorSaturationShouldBePinned() {
+    assertThat(Color.HSVToColor(new float[]{0f, -1.0f, 0.5f})).isEqualTo(0xff808080);
```
Comment (utzcoz) [inline robolectric/src/test/java/org/robolectric/shadows/ShadowColorTest.java:86]: Maybe we also can add these tests to ctesque.

**Comment by utzcoz** [comment]
@mlebedy Please keeping one commit for your PR always. I don't recommend to use GitHub Webpage button to update your code base.

**Comment by mlebedy** [comment]
Java Color#HSVToColor:

https://cs.android.com/android/platform/superproject/+/refs/heads/master:frameworks/base/graphics/java/android/graphics/Color.java;drc=7346c436e5a11ce08f6a80dcfeb8ef941ca30176;l=1468

It calls the native method defined here:

https://cs.android.com/android/platform/superproject/+/refs/heads/master:frameworks/base/libs/hwui/jni/Shader.cpp;drc=7346c436e5a11ce08f6a80dcfeb8ef941ca30176;l=42

Which calls the real implementation within the Skia library:

https://cs.android.com/android/platform/superproject/+/refs/heads/master:external/skia/src/core/SkColor.cpp;drc=7346c436e5a11ce08f6a80dcfeb8ef941ca30176;l=74

**Comment by utzcoz** [comment]
@mlebedy Thanks for your contribution. And configuration to your first PR for Robolectric.

File: robolectric/src/test/java/org/robolectric/shadows/ShadowColorTest.java (hunk 1 above)
Comment (utzcoz) [reply robolectric/src/test/java/org/robolectric/shadows/ShadowColorTest.java]: @mlebedy if you like, you also can add these tests to ctesque module for Emulator testing.

Comment (mlebedy) [reply robolectric/src/test/java/org/robolectric/shadows/ShadowColorTest.java]: OK! Here is my PR for the integration test: https://github.com/robolectric/robolectric/pull/8281

**Pull request merged**